src/DataWrangling/ETOPO/ETOPO.jl

```diff
@@ -0,0 +1,84 @@
+module ETOPO
+
+using Scratch
+export ETOPOBathymetry
+
+import ClimaOcean.DataWrangling:
+    metadata_filename,
+    default_download_directory,
+    metadata_path,
+    Metadata,
+    Metadatum,
+    all_dates,
+    first_date,
+    last_date,
+    download_dataset,
+    download_progress
+    longitude_interfaces,
+    latitude_interfaces,
+    is_three_dimensional,
+    reversed_vertical_axis
+
+using Downloads
+
+using Oceananigans
+using Oceananigans.DistributedComputations: @root
+
+download_ETOPO_cache::String = ""
+function __init__()
+    global download_ETOPO_cache = @get_scratch!("ETOPO")
+end
+
+ETOPO_bathymetry_variable_names = Dict(
+    :temperature => "temperature",
+    :salinity    => "salinity"
+)
+
+struct ETOPOBathymetry end
+
+default_download_directory(::ETOPOBathymetry) = download_ETOPO_cache
+reversed_vertical_axis(::ETOPOBathymetry) = true
+longitude_interfaces(::ETOPOBathymetry) = (-180,180)
+latitude_interfaces(::ETOPOBathymetry) = (-90,90)
+is_three_dimensional(::ETOPOBathymetry) = false
```

Comment:
For some reason, Julia doesn't 'see' this method or the two above it... I think I am importing and exporting correctly, especially because `all_dates` seems to work below

Reply:
So it shows 

```Julia
julia> is_three_dimensional(ETOPOmetadata)
true
```

when it should be `false`

Reply:
the way you defined it here, `is_three_dimensional` expects a dataset as argument, not a metadata.

Reply:
where is this method used? is it used to build the grid?

Reply:
Fixed this now - I no longer use `is_three_dimensional`